docs/build/eps.md

```diff
@@ -802,3 +802,32 @@ See more information on the CANN Execution Provider [here](../execution-provider
 
 * The CANN execution provider supports building for both x64 and aarch64 architectures.
 * CANN excution provider now is only supported on Linux.
+
+## Cloud
+
+See the [Cloud Execution Provider](../execution-providers/Cloud-ExecutionProvider.md) documentation for more details.
+
+### Prerequisites
+
+For Linux, before bulding please:
+
+* install openssl dev package into the system, which is openssl-dev for redhat and libssl-dev for ubuntu.
+* if got multiple openssl dev versions installed in the system, please set environment variable "OPENSSL_ROOT_DIR" to the desired version, for example:
```

Comment:
should be: "if you have multiple versions"

Reply:
Done.